draft-ietf-scitt-architecture.md

```diff
@@ -482,25 +482,23 @@ Might dereference to:
 }
 ~~~
 
-### Naming Artifacts
+### Support for Multiple Artifacts
 
-Many Issuers issue Signed Statements about different Artifacts under the same DID, so it is important for everyone to be able to immediately recognize by looking at the Envelope of a Signed Statements what Artifact it is referring to.
+Many Issuers produce Signed Statements about various Artifacts under the same Identity. Issuers and Verifiers must be able to recognize the Artifact to which the statements pertain by looking at the Signed Statement.
```

Comment:
I don't agree that the discriminator for a product is within the signed statements.  In the case where product identity is mandated within the artifact, requiring it to be replicated into the detached signature is unclear.  
In the use case where there is a global or major Transparency service, I could make a case that the product information hints to auditors more than the frequency of which statements are process, and that may be unacceptable.  Right now the information disclosure in a signed detached signature is limited.  Can someone make a case why we must\should scope this here?  I could see it being necessary for some vertical process.